developer_documentation/rcdgen_markup.rst

```diff
@@ -873,9 +877,22 @@ Attributes:
 - ``logo`` - FreeRCT logo sprite.
 - ``new_game`` - New Game button sprite.
 - ``load_game`` - Load Game button sprite.
+- ``launch_editor`` - Scenario Editor button sprite.
 - ``settings`` - Settings button sprite.
 - ``quit`` - Quit button sprite.
 
+MISN
+~~~~
+Attributes:
+
+- ``internal_name`` - Internal name of the mission.
+- ``texts`` - strings_ node containing the mission's texts.
+- ``max_unlock`` - Maximum number of unlocked unsolved scenarios in the mission.
+- ``scenarios`` - Total number of scenarios in the mission.
+- ``file_N`` - Path to the FCT file for the N-th scenario.
```

Comment:
Should we have requirements wrt to "path"?

- is it an absolute path or a relative path? If the latter, relative to what?
- Kind of directory separators? (unless we have path strings already, but I don't think they exist yet in the rcd files.)